src/status_im/contexts/onboarding/create_password/view.cljs

```diff
@@ -119,102 +130,70 @@
                 {:content create-password-doc
                  :shell?  true}]))
 
-(defn- navigate-back
-  []
-  (rf/dispatch [:navigate-back]))
+(defn- navigate-back [] (rf/dispatch [:navigate-back]))
 
 (defn- page-nav
-  []
-  (let [{:keys [top]} (safe-area/get-insets)]
-    [quo/page-nav
-     {:margin-top top
-      :background :blur
-      :icon-name  :i/arrow-left
-      :on-press   navigate-back
-      :right-side [{:icon-name :i/info
-                    :on-press  on-press-info}]}]))
+  [top]
+  [quo/page-nav
+   {:margin-top top
+    :background :blur
+    :icon-name  :i/arrow-left
+    :on-press   navigate-back
+    :right-side [{:icon-name :i/info
+                  :on-press  on-press-info}]}])
+
+(defn- help-and-confirm-button
+  [{:keys [password-validations same-passwords? on-submit]}]
+  (let [{customization-color :color} (rf/sub [:onboarding/profile])
+        all-requirements-met?        (and (:non-empty? password-validations)
+                                          (:long-enough? password-validations)
+                                          (:short-enough? password-validations)
+                                          same-passwords?)]
+    [rn/view {:style style/footer-container}
```

Comment:
Moved this `and` to here from its parent. Its siblings don't use this value, by moving this closer to its usage we reduce the mental overhead while understanding the parent component